Equinox/SimpleTimer.h

```diff
@@ -45,20 +45,13 @@ class SimpleTimer
 		pausedTicks = 0;
 	}
 
-	unsigned int GetTimerTicks() const
+	unsigned int Read() const
```

Comment:
What happens is you call Read when the timer is stopped? you must return the time when it was stopped. Here you are returning 0 if the timer is stopped such is not correct.